src/js/components/search/table/cells/ResultsTableGenericCell.jsx

```diff
@@ -32,6 +33,14 @@ export default class ResultsTableGenericCell extends React.Component {
             rowClass += ' last-column';
         }
 
+        if (this.props.column === 'award_id') {
```

Comment:
The point of this component (per the component name) is to be generic. Instead of adding column-specific logic to the component, I would recommend creating a new component specifically for the ID column. See `ResultsTable` notes for more implementation details.